tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -657,6 +663,13 @@ class MklConvOp : public OpKernel {
 
       // TODO(mdfaijul): Extend the basic parameters for data types and fusions
       this->ExtendConvFwdParams(context, convFwdDims);
+#ifdef DNNL_AARCH64_USE_ACL
+      // Specifics of ACL: a primitive per constant weights ptr
+      convFwdDims.filter_address = const_cast<void*>(
+                                   static_cast<const void*>(
+                                   filter_tensor.flat<Tfilter>().data()));
+#endif
```

Comment:
What is the rationale for adding the filter address to the key? If the filter is not a constant, the address could change from iteration to iteration.

Reply:
@agramesh1, this is due to Compute Library programming model, it creates a primitive per filter tensor, not per shape. And during the inference there is usually a set of constant addresses for the weights.

Reply:
ok. then it should be fine.